recipes-debian/openldap/openldap/initscript

```diff
@@ -0,0 +1,35 @@
+#! /bin/sh
```

Comment:
This initscripts isn't enough to start debian version of slapd. To start slapd via this script, you'll get following error.

```
root@qemuarm64:/etc/init.d# ldapsearch -LLL -Y EXTERNAL -H ldapi:/// -b cn=config -d 7
ldap_url_parse_ext(ldapi:///)
ldap_create
ldap_url_parse_ext(ldapi:///??base)
ldap_sasl_interactive_bind: user selected: EXTERNAL
ldap_int_sasl_bind: EXTERNAL
ldap_new_connection 1 1 0
ldap_int_open_connection
ldap_connect_to_path
ldap_new_socket: 3
ldap_connect_to_path: Trying /var/run/slapd/ldapi
ldap_connect_timeout: fd: 3 tm: -1 async: 0
ldap_ndelay_on: 3
ldap_close_socket: 3
ldap_msgfree
ldap_err2string
ldap_sasl_interactive_bind_s: Can't contact LDAP server (-1)
```
The /var/run/slapd/ldapi is a unix domain socket file which slapd creates during its boot process. However, there is no /var/run/slapd directory in root system so that slapd cannot create ldapi file. The debian version of init script creates /var/run/ldapi during startup process ([debian/slapd.init](https://salsa.debian.org/openldap-team/openldap/-/blob/master/debian/slapd.init#L129)). So please create /var/run/ldap directory in the init script.